terms.html

```diff
@@ -105,7 +102,7 @@
   </dd>
   <dt><dfn data-lt="verifier|verifiers|verifier's|credential verifiers|credential verifier's">verifier</dfn></dt>
   <dd>
-An <a>entity</a> that receives one or more <a>profiles</a> for
+An <a>entity</a> that receives one or more <a>verifiable profiles</a> for
```

Comment:
Deutche Telekom requested that not all credentials and profiles are verifiable. This was a reasonable request. Changing the language to "verifiable profile" removes that distinction, which may just result in them formally objecting to the spec if it goes forward in that way. There was a nuance in the spec before that doesn't exist now, we lose functionality in a bad way.

Reply:
I dont think it is a reasonable request to make of the Verifiable Credentials working group, that credentials dont need to be verifiable. This surely broadens the scope too much.

Reply:
@David-Chadwick,

> I dont think it is a reasonable request to make of the Verifiable Credentials working group, that credentials dont need to be verifiable. This surely broadens the scope too much.

I recommend taking this argument to Deutche Telekom, who has the power to formally object. If they are fine with it, that's great. If not, I don't know how much we gain by fighting over it.

Reply:
@AxelNennker and Jörg Heuer at Deutche Telekom.

Reply:
btw, I also agree that there is no reason to insist that there be a proof on the credential.

Reply:
A verifier must always be able to trust the credentials it receives. This is written into the trust model. 

Trust in the credential can be established in two ways:
a) it can be done by using a trusted channel between the issuer and the verifier, and transferring an unsigned credential along a trusted channel, or 
b) it can use an untrusted channel and a credential signed by the issuer. 

I do not see how the VC data model, which requires the VC to be passed from the trusted issuer to an untrusted user, can then establish a trusted channel between the verifier and the untrusted user. Therefore it is essential that for the VC data model, all credentials have proofs and are verifiable.